service-api/pom.xml

```diff
@@ -48,5 +48,19 @@
             <groupId>org.springframework</groupId>
             <artifactId>spring-context</artifactId>
         </dependency>
+        <dependency>
+            <groupId>io.jsonwebtoken</groupId>
+            <artifactId>jjwt</artifactId>
+            <version>0.9.1</version>
```

Comment:
Please, move version to properties section

Reply:
Done